src/main/java/us/daconta/xlmeco/GrpcXlmServer.java

```diff
@@ -26,7 +26,7 @@ private static Properties loadProperties(String fileName) throws IOException {
 
     public static void main(String[] args) throws Exception {
         // Load properties from config file
-        Properties properties = loadProperties("config.properties");
+        Properties properties = loadProperties("config.properties.template");
```

Comment:
You don't load the template.  Need to change this back to config.properties.  The template is only used to copy it to the config.properties file and add your API keys.  I actually have my "real" file stored away in another directory and I just copy it in when I want to test.